### DISCUSSION TITLE
OSM Conflator tool - is it no longer maintained/needed/used?

### DISCUSSION THREAD

**Discussion by matkoniecz** [organicmaps/organicmaps#8419] [Q&A] [answered]
I am going through tools listed at https://wiki.openstreetmap.org/wiki/Conflation (as I am looking for tool to solve problem that I have, ideally without starting a new project).

I noticed https://wiki.openstreetmap.org/wiki/OSM_Conflator that was made as part of MAPS.ME and seems to not be part of Organic Maps project.

Is it finished? No longer needed? Abandoned? Something else?

Would it be recommended in at least some use cases or is it wholly replaced by other software?

**Comment by biodranik** [comment, marked as answer]
The tool was created by @Zverik , you can maintain/develop it if you want. I've made a fork for OM just in case: https://github.com/organicmaps/osm_conflate

**Reply by matkoniecz** [reply]
I guess my main question now is whether @Zverik would recommend avoiding its use or suggest some alternative.

**Comment by Zverik** [comment]
OSM Conflator should work, and I see people sneakily doing some small imports (validating them with CF Audit). The only issue with it is that it is not documented well enough, but it is definitely not obsolete. I still consider it the best import tool for OSM there is.

The reason it's not been developed for five years is, I ironed out most issues when doing multiple imports during my maps.me time.
